core/src/test/java/com/sequenceiq/cloudbreak/converter/v4/stacks/cluster/ClusterToClusterV4ResponseConverterTest.java

```diff
@@ -1,4 +1,4 @@
-package com.sequenceiq.cloudbreak.converter.stack.cluster;
+package com.sequenceiq.cloudbreak.converter.v4.stacks.cluster;
 
 import static org.junit.Assert.assertEquals;
```

Comment:
Could you please migrate the class to JUnit 5?

Reply:
done

Reply:
Sorry, this does not seem to be done. It is still based on JUnit 4.

Reply:
I added new file with test in CloudStorageConverterTest, and it is JUnit 5 now.

I added new tests in the existing file ClusterToClusterV4ResponseConverterTest, which uses Junit 4. Do you want to change this to JUnit 5 as well? that would requires me to change existing tests. I can do it.

Reply:
done

Reply:
Super, thanks! 👍